### ISSUE TITLE
UX minor issue: in dropdown of components that allows copying settings from another component, the edited component is displayed too which has not sense (as I understand copying from itself is senseless)

### ISSUE THREAD

**Issue by kamilrafik** [vdm-io/Joomla-Component-Builder#900]
UX minor issue: in dropdown of components that allows copying settings from another component, the edited component is displayed too which has not sense (as I understand copying from itself is senseless). Please, see screenshot.

![Screenshot from 2022-04-17 13-09-39](https://user-images.githubusercontent.com/4179340/163710062-67d60e8b-3d42-41d1-b1a6-e9ff8965956c.png)

**Comment by Llewellynvdm** [comment]
Good point, I actually thought that I did take care of that.... This is a bug, so again thanks for this issue, we need this fixed.

The field type is the normal [joomlacomponent](https://github.com/vdm-io/Joomla-Component-Builder/blob/staging/admin/models/forms/component_dashboard.xml#L110) and the code for that field type is [found here](https://github.com/vdm-io/Joomla-Component-Builder/blob/staging/admin/models/fields/joomlacomponent.php#L141)
